docs/administration/parameters.md

```diff
@@ -1 +1,23 @@
-# Platform
\ No newline at end of file
+# Platform
+
+## Platform Announcement
+
+This section gives you the possibility to set and display Announcements in the platform. Those announcement will be
+visible to every user in the platform, on top of the interface.
+
+They can be used to inform all your users' community of important information, like a scheduled downtime, an incoming
+upgrade, or even an important tips regarding the usage of the platform.
+
+![Broadcast message configuration](assets/broadcast_message_configuration.png)
+
+An Announcement can be accompanied by a “Dismiss” button. When click by a user, it makes the message disappeared
+for this user.
+
+![Broadcast message display dismissible](assets/broadcast_message_display_dismissible.png)
```

Comment:
It seems the screenshots are not updated with the last style modifications (but maybe it can be done after the PR has been merged)

Reply:
Indeed, thank you for your vigilance !
It's updated now